packages/accounts-controller/src/AccountsController.ts

```diff
@@ -78,7 +78,7 @@ export type AccountsControllerGetAccountAction = {
   handler: AccountsController['getAccount'];
 };
 
-export type AllowedAccountsControllerActions =
```

Comment:
Nit: We should list the removal of this as a breaking change. I agree that it doesn't need to be exported though, and likely isn't used.

Likewise for the allowed events export

Reply:
The `AllowedAccountsController{Actions,Events}` types were only introduced in a PR merged yesterday: https://github.com/MetaMask/core/pull/4021, so they were never released. I think that makes it safe not to list them on the changelog, since there ends up being no user-facing changes from the previous release.

Reply:
Ah thanks, I had not realized they were new. Makes sense!